### PR TITLE
Optimisation: simplify some of the LCD fullscreen code

### PR DESCRIPTION
**Pull request by gudnimg**
Removed only lines of code that shouldn't have any effect on the UI for the user. Mostly just redundant code.

Change in memory:
Flash: -62 bytes
SRAM: 0 bytes

### CODE REVIEW

File: Firmware/ultralcd.cpp [hunk 1]
```diff
@@ -2992,23 +2992,15 @@ static const char* lcd_display_message_fullscreen_nonBlocking_P(const char *msg)
 {
     lcd_set_cursor(0, 0);
     const char *msgend = msg;
-    uint8_t row = 0;
     bool multi_screen = false;
-    for (; row < 4; ++ row) {
-        while (pgm_is_whitespace(msg))
```
Comment (leptun) [inline Firmware/ultralcd.cpp:2998]: I'm not sure it's ok to remove this. What if the word on the previous line finishes with the last character on the last column of the screen? Wouldn't that cause the next line to have a useless space at the beginning?

Comment (gudnimg) [reply Firmware/ultralcd.cpp:2998]: You're correct. See images below:

`lcd_display_message_fullscreen_P(_i("Averyveryverlongword Press the knob to pr eheat nozzle and continue."));`

MK3 branch test:
IMG_2731


PR branch test:
IMG_2729

Comment (gudnimg) [reply Firmware/ultralcd.cpp:2998]: Fixed in aca9e79

File: Firmware/ultralcd.cpp [hunk 2]
```diff
@@ -3198,9 +3192,6 @@ uint8_t lcd_show_multiscreen_message_with_choices_and_wait_P(
             if (allow_timeouting && _millis() - previous_millis_cmd > LCD_TIMEOUT_TO_STATUS) {
                 return LCD_BUTTON_TIMEOUT;
             }
-            manage_heater();
-            manage_inactivity(true);
```
Comment (DRracer) [inline Firmware/ultralcd.cpp:3202]: Why was this removed? Is lcd_show_multiscreen_message_with_choices_and_wait_P always accompanied with some other ways of managing heaters and inactivity?

Comment (gudnimg) [reply Firmware/ultralcd.cpp:3202]: There is a call to `delay_keep_alive(50)` above. It calls these functions. There is no need to do it twice, that just increases code size.

Comment (DRracer) [reply Firmware/ultralcd.cpp:3202]: :+1:

**Review by DRracer — APPROVED** [review]
